### PR TITLE
Bump up KubeVirt version to 0.7.0-alpha.3

### PR DESCRIPTION
**Pull request by lukas-bednar**
Signed-off-by: Lukas Bednar <lbednar@redhat.com>

### CODE REVIEW

**Comment by lukas-bednar** [comment]
@dcritch It is not really good to keep kubevirt.yaml.j2 in this repository it is changing between releases ... we need to make sure that we take it from kubevirt.

File: roles/kubevirt/templates/kubevirt.yaml.j2 [hunk 1]
```diff
@@ -509,1527 +500,10 @@ spec:
...
-                            apic:
-                              properties:
-                                enabled:
-                                  description: Enabled determines if the feature should
-                                    be enabled or disabled on the guest Defaults to
-                                    true
-                                  type: boolean
-                                endOfInterrupt:
-                                  description: EndOfInterrupt enables the end of interrupt
-                                    notification in the g
```
Comment (lukas-bednar) [inline roles/kubevirt/templates/kubevirt.yaml.j2]: this is failing CI

**Comment by lukas-bednar** [comment]
Tests are not updated to new version of kubevirt ...
```
  Test Panicked

  the server could not find the requested resource (delete offlinevirtualmachines.kubevirt.io)

  /gimme/.gimme/versions/go1.10.linux.amd64/src/runtime/panic.go:505
```

**Comment by shiywang** [comment]
I was just wondering the this kind of bumping version pr bump every releases kubevirt released or just stable version ? 

Also if this is not in a hurry, I guess we can let my integration/e2e tests merged, then it will cover the kubevirt && CDI compatibility at least

**Comment by lukas-bednar** [comment]
@shiywang we usually does with releases (not with pre-releases), but unfortunately we need to move on with presets [issue](https://github.com/kubevirt/kubevirt-ansible/pull/262) which was fixed in 0.7.0-alpha.3.

What is your estimation, that your integration tests gets in ?

**Comment by shiywang** [comment]
@lukas-bednar okay, thanks for the explain, I guess this pr is only for fixed problem in #262.
my cdi tests depend on couple other prs and also glusterfs enabled in oVirt CI, so I don't know, but I hopefully I could push the progress a little bit make it merge as soon as possible, I'll talk to Adam today

**Comment by rthallisey** [comment]
offline virtual machines don't exist anymore.  The tests need to look for the new resource.
```
------------------------------

Panic [0.015 seconds]

[AfterSuite] AfterSuite 

/root/go/src/kubevirt.io/kubevirt-ansible/tests/tests_suite_test.go:21


  Test Panicked

  the server could not find the requested resource (delete offlinevirtualmachines.kubevirt.io)

  /gimme/.gimme/versions/go1.10.linux.amd64/src/runtime/panic.go:505


  Full Stack Trace

  	/gimme/.gimme/versions/go1.10.linux.amd64/src/runtime/panic.go:505 +0x229

  kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests.PanicOnError(0x1211080, 0xc420230240)

  	/root/go/src/kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests/utils.go:645 +0x4a

  kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests.cleanNamespaces()

  	/root/go/src/kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests/utils.go:553 +0x261

  kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests.AfterTestSuitCleanup()

  	/root/go/src/kubevirt.io/kubevirt-ansible/vendor/kubevirt.io/kubevirt/tests/utils.go:292 +0x27

  kubevirt.io/kubevirt-ansible/tests_test.glob..func3()

  	/root/go/src/kubevirt.io/kubevirt-ansible/tests/tests_suite_test.go:22 +0x20

  kubevirt.io/kubevirt-ansible/vendor/github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync(0xc4203d4fc0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)

  	/root/go/src/kubevirt.io/kubevirt-ansible/tests/tests_suite_test.go:14 +0x64

  testing.tRunner(0xc4202cc4b0, 0x11a6b60)

  	/gimme/.gimme/versions/go1.10.linux.amd64/src/testing/testing.go:777 +0xd0

  created by testing.(*T).Run

  	/gimme/.gimme/versions/go1.10.linux.amd64/src/testing/testing.go:824 +0x2e0
```

**Comment by dcritch** [comment]
@lukas-bednar "It is not really good to keep kubevirt.yaml.j2 in this repository it is changing between releases ... we need to make sure that we take it from kubevirt."

That has always been the goal, but it is taking a bit longer than we expected to get kubevirt to supply it so we can stop shipping it [0]. In the meantime, I ran the build-manifest.sh in that PR to generate it the last time.

[0]: https://github.com/kubevirt/kubevirt/pull/1116

**Comment by shiywang** [comment]
so I guess only update vendor will pass

**Comment by lukas-bednar** [comment]
Closing in favor of #295

**Pull request closed without merging**